ONSdigital/census-rm-monitoring
# Add monitor for bad messages

# Motivation and Context

We would like to update the bad message metric to alert by queue
# What has changed

New function to grab bad message summaries from exception manager
# How to test?



Terraform your env using the branch attached to the card. Build the rabbit monitor docker image using this branch and push to your GCP env. Deploy using the rabbit monitor manifest in optional on the census-rm-kubernetes branch attached to the card. Put bad messages on to rabbit and check they are logged in the queue stats in rabbit monitor
# Links



https://trello.com/c/XYJL16hT/2121-update-bad-message-alerts-8
